### DISCUSSION TITLE
How to uninstall Quarto on MacOS?

### DISCUSSION THREAD

**Discussion by iabraham** [quarto-dev/quarto-cli#10164] [Q&A]
### Description

How does one uninstall Quarto on MacOS? I would like to completely remove and undo any changes made by the `.pkg` during an installation but I can't seem to find the instructions anywhere. 

Does it simply amount to deleting the `/Applications/quarto` folder?  What about any symlinks and/other libraries?

**Comment by mcanouil** [comment]
Removing it should be enough as Quarto should not make anything special in your environment and Quarto uses embedded version of libraries so no linking outside of it.

**Reply by cscheid** [reply]
I think our installer might create a `/usr/local/bin/quarto` symlink; it's worth checking.

**Reply by tdstein** [reply]
An entry is also added to /etc/paths.d in the [postinstall script](https://github.com/quarto-dev/quarto-cli/blob/4f4132844b5764d21d188ad9cba52a0fd5917a17/package/scripts/macos/pkg/postinstall#L6), which you can remove with `rm  /etc/paths.d/quarto`.
